packages/react-components/react-tags/src/components/Tag/renderTag.tsx

```diff
@@ -13,7 +13,7 @@ import { AvatarContextProvider } from '@fluentui/react-avatar';
 export const renderTag_unstable = (state: TagState, contextValues: TagContextValues) => {
   const { slots, slotProps } = getSlotsNext<TagSlots>(state);
 
-  return (
+  return state.dismissed ? null : (
```

Comment:
Does this make any context in the sense of Tags being used without a TagGroup/TagList?

```tsx
<Tag dismissed />
```

The above 👆👆👆 just renders null and the user can do the same with `onClick` and some state 😐

Then if we implement this as a collection we get this

```tsx
<TagGroup>
  {tags.map(tag => <Tag dismissed={dismissed.includes(tag) onDismiss={createDismissHandler(tag)}} />)}
</TagGroup>
```

The above isn't that obvious to memoize because of the handler factory and also isn't consistent with our selection components.

Considering that all `dismissed` does as a property is render null...is it really worth having? users can simply choose not to render that tag.

If we wanted easier group handling we could do something like our selection APIs: 

* https://react.fluentui.dev/?path=/docs/components-menu-menu--default#checkbox-items
* https://react.fluentui.dev/?path=/docs/components-radiogroup--default#controlled-value
* https://react.fluentui.dev/?path=/docs/components-tablist--default
* https://react.fluentui.dev/?path=/docs/components-select--default#controlled
* https://react.fluentui.dev/?path=/docs/components-combobox--default#controlled

Reply:
I get that we might want a group component for keyboard navigation, but I really do wonder if we need to handle dismissing for users, since all it does it stop rendering the component